scripts/mutect2_wdl/mutect2.wdl

```diff
@@ -98,6 +98,47 @@ task SplitIntervals {
   }
 }
 
+task CollectSequencingArtifactMetrics {
+    String gatk4_jar
+    File bam_file
+    String output_prepend
+    File ref_fasta
+    Boolean run_orientation_bias_filter
+
+    command {
+        if [ ${run_orientation_bias_filter} = true ]; \
```

Comment:
Why run this task at all if you are not going to do anything?  Move the conditional to the workflow logic.  Use the new WDL syntax for that.  *From now on, do not make tasks that implement the conditional logic nor should you make tasks that are no-ops*  These can be very expensive.

https://software.broadinstitute.org/wdl/userguide/topic?name=wdl-plumbing